cypress/page/products-list.page.ts

```diff
@@ -3,9 +3,8 @@ class ProductsListPage {
   private proceedCheckoutBtn: string;
 
   constructor() {
-    this.addCartBtn =
-      '#center_column a.button.ajax_add_to_cart_button.btn.btn-default';
-    this.proceedCheckoutBtn = '#layer_cart a';
+    this.addCartBtn = '.button-container .ajax_add_to_cart_button';
```

Comment:
here you can use an attribute to indicate we are referring to the "Add to cart" button